client/src/components/utils/Footer/Footer.js

```diff
@@ -4,7 +4,7 @@ import { Anchor, Footer as SiteFooter, Text } from "grommet";
 export default function Footer() {
   return (
     <SiteFooter
-      style={{ padding: 20, position: "fixed", bottom: 0, width: "100vw" }}
+      style={{ padding: 20, position: "static", bottom: 0, width: "100vw" }}
```

Comment:
This should probably be reverted back to "fixed" cuz 
![image](https://user-images.githubusercontent.com/31177376/114089908-bd105000-9884-11eb-8bb8-e9740720a2a1.png)